openstack/clustering/v1/policies/requests.go

```diff
@@ -54,3 +54,46 @@ func List(client *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pa
 		return p
 	})
 }
+
+// CreateOpts params
+type CreateOpts struct {
+	Name string `json:"name"`
+	Spec Spec   `json:"-"`
+}
+
+type Spec struct {
```

Comment:
With this `Spec` type created, does it make sense to change the `Policy` struct's `Spec` field to be of type `Spec` rather than `map[string]interface{}`? Will a Spec ever return more fields than Description, Properties, Type, and Version?

Reply:
Done.  The same Spec struct should be used for the request and return.